cmd/login.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"github.com/redhat-developer/odo/pkg/auth"
+	"github.com/spf13/cobra"
+)
+
+var (
+	Username string
+	Password string
+	Token    string
+	CAAuth   string
+	SkipTLS  bool
+)
+
+// versionCmd represents the version command
+var loginCmd = &cobra.Command{
+	Use:   "login",
+	Short: "Login to cluster",
+	Long:  "Login to cluster",
```

Comment:
Will it be better to explain what args are mandatory/mutually exclusive etc... here ?

Reply:
nothing is mandatory, I have updated the example section